blueprints/ember-cli-fastboot/index.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-env node */
+const recast = require('recast');
+const { readFileSync, writeFileSync } = require('fs');
+
+module.exports = {
+  description: '',
+  normalizeEntityName() {
+    // no-op
+  },
+
+  afterInstall() {
+    let targetsFile = './config/targets.js'
```

Comment:
This path is not guaranteed to be the correct location for targets. We should follow the logic in ember-cli for this:

https://github.com/ember-cli/ember-cli/blob/614c854171d64375ef904861119e9e2d69f75d9f/lib/models/project.js#L275-L281